api/db-cdr/changelog-schema/db.changelog-7.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+  <changeSet author="brianfreeman" id="changelog-7">
+
+    <createTable tableName="db_domain">
+      <column name="domain_id" type="varchar(20)">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="domain_display" type="VARCHAR(30)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="domain_desc" type="VARCHAR(4000)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="domain_parent" type="varchar(20)">
+        <constraints nullable="false"/>
+      </column>
+      <column name="domain_route" type="VARCHAR(80)">
+        <constraints nullable="true"/>
+      </column>
+    </createTable>
+
```

Comment:
Do this table need any foreign key constraints?

Reply:
No. Custom table. Somewhat like omop domain but used to hold content for the home page.